app.js

```diff
@@ -14,6 +14,17 @@ app.get('/', (req, res) => {
   res.send('Hello from Artёm!');
 });
 
+app.get('/task/full-info-stud', (req, res) => {
+  taskApi
+    .getAssignmentById(req.query.assId)
+    .then((assignment) => {
+      res.status(200).send(assignment);
+    })
+    .catch((err) => {
+      res.status(400).send(err);

```

Comment:
400? @AntonGrigoriev what do you think?

Reply:
I wish to offer to add 204(no content). FE shouldnt check array that comes from backend i think.